content/community/code-reviews.md

```diff
@@ -17,3 +17,24 @@ Here are some of our tips for conducting *Code Reviews* the SweetOps way. If you
 6. Use ChatOps commands like `/test all`, `/test bats`, `/test readme`, `/test terratest` to run integration tests
 7. Recommend changes to better conform to our best-practices.
 8. Quote comments you're replying to make your responses more clear.
+
+
+
+### Specifics for Terraform modules
+
+We use automated testing to enforce certain standards for our Terraform modules. Currently these are run via GitHub Actions, and you can look at the logs of failing tests by clicking the `Details` link in the PR status list. Here is a partial list of rules that are enforced:
+
+- All modules referenced must be pinned to an exact, numbered version. Cannot be `master` or a range like `>= 0.9.0`
+- All providers must have version pinning of the form `>=` (can be `>= x.x` or `>= x.x.x`). More restrictive pinning is not allowed.
+- All modules that no longer support Terraform versions older than 0.12.26 must be upgraded to refer to providers using Terraform Registry format (explicit `source` field). 
+- All modules must have their `README` updated to the current standard. **Note:** `README.md` is generated by tooling from `README.yaml`. Anything you want to update in the `README` must be updated in `README.yaml` or it will simply be overwritten. Updating `README` is usually nothing more that regenerating it.
```

Comment:
```suggestion
- All modules must have their `README` updated to the current standard. **Note:** `README.md` is generated by tooling from `README.yaml`. Anything you want to update in the `README` must be updated in `README.yaml` or it will simply be overwritten. Updating `README` is usually nothing more that regenerating it.
```
```suggestion
- All modules must have their `README` updated to the current standard. **Note:** `README.md` is generated by tooling from `README.yaml`. Anything you want to update in the `README` must be updated in `README.yaml` or it will simply be overwritten. Updating the `README` is usually requires nothing more than regenerating it.
```